anycloud/cli/src/deploy/mod.rs

```diff
@@ -587,10 +624,43 @@ pub async fn edit_deploy_config() {
       .with_initial_text(config.vmType.to_string())
       .interact_text()
       .unwrap();
+    let min_replicas: Option<u32>;
```

Comment:
same as above - i'm not sure why `min_replicas` is an `Option` anyways? it's never set to `None`...

Reply:
Add it as `Option` to have "backwards compatibility", if in your current config theres no minReplicas or maxReplicas still works. Thats why I added it but yes, initially I thought it as pure `u32`, not and `Option`.

Reply:
More than backwards compatibility, it is an optional parameter so it parsing `anycloud.json` should not fail if it is missing.